wasm/read.go

```diff
@@ -35,6 +35,17 @@ func readBytes(r io.Reader, n int) ([]byte, error) {
 	return nil, io.ErrUnexpectedEOF
 }
 
+func WriteByte(w io.Writer, b byte) error {
```

Comment:
perhaps put it in a new `wasm/write.go` file?

also, perhaps:
```suggestion
func writeByte(w io.Writer, b byte) error {
```

Reply:
The `ReadByte` method is used in `disasm` package. For the sake of symmetry, I exposed it too. and maybe it will be used in later fix.

Reply:
fixed